server/game/cards/locations/07/blackbetha.js

```diff
@@ -0,0 +1,57 @@
+const DrawCard = require('../../../drawcard.js');
+
+class BlackBetha extends DrawCard {
+    //Just used Iron Fleet Scout as template.  Overly verbose, but easy to understand.  Could be refactored into something more concise (a la The Valyrian).
+
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: card => card.name === 'Ser Davos Seaworth',
+            effect: ability.effects.addKeyword('renown')
+        });
+        this.action({
+            title: 'Kneel to give attacking character +X STR',
+            cost: ability.costs.kneelSelf(),
+            condition: () => this.game.currentChallenge && this.game.currentChallenge.attackers.length >= 1,
+            method: 'kneel'
```

Comment:
I don't think this matters much, but I believe using a handler instead of a method is preferred for actions now.